README.md

```diff
@@ -14,169 +42,123 @@ My first project at Microverse
   - [Install](#install)
   - [Usage](#usage)
   - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
+  - [Deployment](#deployment)
 - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Hello-microverse-project] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 Hello Microverse! <a name="about-project"></a>
 
-**[Hello-microverse-project]** is my First Project at Microverse
+> This is my first project in Microverse using Linters and Feature Branches, HTML, CSS and more...
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
+1- HTML.
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
+2- CSS.
 
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
+### Tech Stack <a name="tech-stack"></a>
 
 <details>
-<summary>Database</summary>
+  <summary>Client</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
   </ul>
 </details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Knowing more about linters to future projects**
+- **Using feature branches to work much better**
+- **Just a simple Hello Microverse! but working on it :)**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+> Comming soon...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+1. Web Browser (Google Chrome, Microsoft Edge, etc).
+2. Code Editor (Visual Studio Code or similar).
+3. Git.
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
+Run this command:
 
 ```sh
   cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone https://github.com/Luiscarlosvd/Hello-microverse-project.git
 ```
---->
 
```

Comment:
- [ ] As mentioned by the previous reviewer and with the showed example about cloning your project locally, kindly make sure you also add the missing command to access the cloned directory before installing linters. .e.
```
  ...
  cd my-desired-folder

  git clone https://github.com/Luiscarlosvd/Hello-microverse-project.git

  cd Hello-microverse-project

   ...
 ```